client/src/app/pages/reports/reports.tsx

```diff
@@ -78,13 +129,54 @@ export const Reports: React.FC = () => {
               <StackItem>
                 <Card>
                   <CardHeader>
-                    <TextContent>
-                      <Text component="h3">{t("terms.currentLandscape")}</Text>
-                    </TextContent>
+                    <Flex>
+                      <FlexItem>
+                        <TextContent>
+                          <Text component="h3">
+                            {t("terms.currentLandscape")}
+                          </Text>
+                        </TextContent>
+                      </FlexItem>
+                      <FlexItem>
+                        <Select
+                          id="select-questionnaires"
+                          isOpen={isQuestionnaireSelectOpen}
+                          selected={selectedQuestionnaire}
+                          onSelect={onSelectQuestionnaire}
+                          onOpenChange={(_isOpen) =>
+                            setIsQuestionnaireSelectOpen(false)
+                          }
+                          toggle={toggleQuestionnaire}
+                          shouldFocusToggleOnSelect
+                        >
+                          <SelectOption key={0} value="All questionnaires">
+                            All questionnaires
+                          </SelectOption>
+                          {questionnaires.map((questionnaire, index) => (
+                            <SelectOption
+                              key={index}
+                              value={questionnaire.name}
```

Comment:
Just curious... Why use `name` here instead of just `id`?

Reply:
@sjd78, normally yes it would make sense to use questionnaire.id for selected value.
Since the questionnaire.name is a unique key constraint there is no reason to go back and forth (see note below)  with the questionnaire id to uniquely identify the questionnaire and pass it along.

Note: Because PF select doesn't allow in the OnSelect() to handle an object, such as Ref, (onSelect) we'd have to get the name of the questionnaire to match the selected value.